_episodes/09-pub-quality-figs.md

```diff
@@ -13,13 +13,66 @@ keypoints:
 - "Publication quality figures are often accompanied by a descriptive caption." 
 ---
 
-`ggplot2` customization for publication quality figures
+Publication quality figures are clean and direct data visualizations that
+clearly illustrate your conclusion to the research question. Publication quality
+figures should include only the necessary data and visualization elements to
+convey the desired message, which are organized by a simple plot frame or theme. 
+All visualization elements are clearly labeled and/or included in a legend and
+are described in the accompanying caption for the graphic. These elements
+include the axis, data shape and color, and any statistical elements giving
+analytical context to the data.
```

Comment:
potentially relevant recommendation to reference:  Rule 9 from [Ten Simple Rules for Better Figures](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003833): Message Trumps Beauty

![image](https://cloud.githubusercontent.com/assets/51236/21570367/3b3ee038-ce7a-11e6-8729-866f38de76be.png)

Figure 8. Message trumps beauty.
This figure is an extreme case where the message is particularly clear even if the aesthetic of the figure is questionable. The uncanny valley is a well-known hypothesis in the field of robotics that correlates our comfort level with the human-likeness of a robot. To express this hypothetical nature, hypothetical data were used () and the figure was given a sketched look (xkcd filter on matplotlib) associated with a cartoonish font that enhances the overall effect. Tick labels were also removed since only the overall shape of the curve matters. Using a sketch style conveys to the viewer that the data is approximate, and that it is the higher-level concepts rather than low-level details that are important [10].